utils/cmut__utils__compile_resource.cmake

```diff
@@ -0,0 +1,139 @@
+
+
+include("${CMAKE_CURRENT_LIST_DIR}/../cmut_message.cmake")
+
+
+function(cmut__utils__get_variable_name VARIABLE_NAME INPUT_FILE)
+    # get the filename and replace all ' ' and '.' with '_'
```

Comment:
this is not what you do in the next string(REGEX ...)

Reply:
done